config/routes.js

```diff
@@ -0,0 +1,12 @@
+const homeRouter = require('../controllers/home');
+const sessionsRouter = require('../controllers/sessions');
+const registrationsRouter = require('../controllers/registrations');
+const companiesRouter = require('../controllers/companies');
+
+
+module.exports = app => {
+  app.use('/', homeRouter);
+  app.use('/sessions', sessionsRouter);
+  app.use('/registrations', registrationsRouter);
+  app.use('/companies', companiesRouter);
+}
```

Comment:
i like the change, but you are not really creating an abstraction between routers and controllers. You just name them like that.

The router should simply call a controller action

```
'GET /companies/ => CompaniesController#IndexAction'
'POST /companies/ => CompaniesController#CreateAction'
'GET /companies/new => CompaniesController#NewAction'
'GET /companies/#{:id} => CompaniesController#ShowAction'
```

In your code your "controller" actions are still just mapping directly to some endpoint